systemtest/src/test/java/io/strimzi/systemtest/kafka/KafkaST.java

```diff
@@ -1735,13 +1644,13 @@ void verifyPresentLabels(Map<String, String> labels, Map<String, String> resourc
         }
     }
 
-    void verifyNullLabels(String[] labelKeys, Map<String, String> labels) {
+    void verifyNullLabels(Set<String> labelKeys, Map<String, String> labels) {
```

Comment:
I don't see it in the code, but you can maybe use List and go through the forEach ? It's just a nit, but when you are changing it 😄 (same for the second method).

But if it would mean that you will have to change other tests, just leave it for now as it is.

Reply:
solution same as in comment lower is the same (I did not notice that this method is used only in part of code i actually deleted as part of this issue), thus I actually removed it as well now.